### DISCUSSION TITLE
Login to Paid Version with Shared Activation Code

### DISCUSSION THREAD

**Discussion by wdfinch** [AtomMaterialUI/material-theme-issues#334] [Q&A] [answered]
Hi there, 

I have a personal JetBrains account and a separate work account. I purchased the premium version of Material Theme on my personal accounts and would like to use the premium version on my work account as well. Right now it seems like I need to login in order to do that. Is there any way I could activate my premium license on my work version by authenticating it with my personal one?

**Comment by wdfinch** [comment, marked as answer]
I was able to resolve this my linking to my personal account.

https://intellij-support.jetbrains.com/hc/en-us/community/posts/4519198498578-Multiple-Jetbrains-Accounts-One-Computer

**Comment by mallowigi** [comment]
I didn't know about that option. Thanks for sharing!
